manifests/apache.pp

```diff
@@ -18,16 +18,16 @@
   ]
 
   apache::vhost { 'katello-ssl':
-    servername          => $::fqdn,
-    serveraliases       => ['katello'],
+    servername          => $certs::apache::hostname,
+    serveraliases       => $certs::apache::cname,
     docroot             => '/var/www',
     port                => 443,
     priority            => '05',
     options             => ['SymLinksIfOwnerMatch'],
     ssl                 => true,
-    ssl_cert            => $certs::ca_cert,
-    ssl_key             => $certs::ca_key,
-    ssl_ca              => $certs::ca_cert,
+    ssl_cert            => $certs::apache::server_cert,
+    ssl_key             => $certs::apache::server_key,
+    ssl_ca              => $certs::apache::default_ca,
```

Comment:
Did you try this? When I tried the installer last week with similar settings, puppet didn't have the values for ::certs::apache::server_cert and server_key for some reason.